src/content/changes/highlights/2023-08-31.md

```diff
@@ -0,0 +1,23 @@
+---
+date: 2023-08-31T14:00:00
+title: Highlights for the week ending August 31 2023
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## Nginx Ingress Controller 
+
+
+## Apps
+
+
+- [external-dns](https://github.com/giantswarm/external-dns-app/) version [v2.39.0](https://github.com/giantswarm/external-dns-app/blob/main/CHANGELOG.md#2390---2023-08-24) replaces monitoring labels with ServiceMonitor CR, adds minAllowed in VPA to avoid OOMs and increases memory limits.
+- [linkerd2-cni](https://github.com/giantswarm/linkerd2-cni-app) version [v1.1.0](https://github.com/giantswarm/linkerd2-cni-app/blob/main/CHANGELOG.md#110---2023-08-29) upgrades to Linkerd v2.13.6.
+- [linkerd-control-plane](https://github.com/giantswarm/linkerd-control-plane-app) version [v1.1.0](https://github.com/giantswarm/linkerd-control-plane-app/blob/main/CHANGELOG.md#110---2023-08-29) upgrades to Linkerd v2.13.6.
+- [linkerd-viz](https://github.com/giantswarm/linkerd-viz-app) version [v1.3.0](https://github.com/giantswarm/linkerd-viz-app/blob/main/CHANGELOG.md#130---2023-08-29), upgrades to Linkerd v2.13.6 and brings better defaults for production environments and pod distribution budgets.
+
+For more Information about the Linkerd v2.13.6 changes, please take a look at the [official release notes](https://github.com/linkerd/linkerd2/releases/tag/stable-2.13.6)
```

Comment:
```suggestion
- [external-dns](https://github.com/giantswarm/external-dns-app/) version [v2.39.0](https://github.com/giantswarm/external-dns-app/blob/main/CHANGELOG.md#2390---2023-08-24) replaces monitoring labels with ServiceMonitor CR, adds minAllowed in VPA to avoid OOMs and increases memory limits.
- [linkerd2-cni](https://github.com/giantswarm/linkerd2-cni-app) version [v1.1.0](https://github.com/giantswarm/linkerd2-cni-app/blob/main/CHANGELOG.md#110---2023-08-29) upgrades to [Linkerd v2.13.6](https://github.com/linkerd/linkerd2/releases/tag/stable-2.13.6).
- [linkerd-control-plane](https://github.com/giantswarm/linkerd-control-plane-app) version [v1.1.0](https://github.com/giantswarm/linkerd-control-plane-app/blob/main/CHANGELOG.md#110---2023-08-29) upgrades to [Linkerd v2.13.6](https://github.com/linkerd/linkerd2/releases/tag/stable-2.13.6).
- [linkerd-viz](https://github.com/giantswarm/linkerd-viz-app) version [v1.3.0](https://github.com/giantswarm/linkerd-viz-app/blob/main/CHANGELOG.md#130---2023-08-29), upgrades to [Linkerd v2.13.6](https://github.com/linkerd/linkerd2/releases/tag/stable-2.13.6) and brings better defaults for production environments and pod distribution budgets.

For more information about the Linkerd v2.13.6 changes, please take a look at the [official release notes](https://github.com/linkerd/linkerd2/releases/tag/stable-2.13.6)
```